lms/static/sass/shared/_header.scss

```diff
@@ -420,8 +424,13 @@
     }
   }
 
+  .logo-header {
```

Comment:
Line 21 has a parent class of header-global
Line 427 has a parent class of header-global-new